wandb/integration/keras/keras.py

```diff
@@ -958,5 +971,31 @@ def _save_model(self, epoch):
         # Was getting `RuntimeError: Unable to create link` in TF 1.13.1
         # also saw `TypeError: can't pickle _thread.RLock objects`
         except (ImportError, RuntimeError, TypeError) as e:
-            wandb.termerror("Can't save model, h5py returned error: %s" % e)
+            wandb.termerror(
+                "Can't save model in the h5py format. The model will be saved as "
+                "W&B Artifacts in the SavedModel format."
+            )
             self.save_model = False
+            self.save_model_as_artifact = True
+
+    def _save_model_as_artifact(self, epoch):
+        if wandb.run.disabled:
+            return
+        try:
+            # Save the model in the SavedModel format.
+            self.model.save(self.filepath[:-3], overwrite=True, save_format="tf")
```

Comment:
This looks good. In fact, we use the same exact technique in the TF/Keras serializer. Can we put a comment in saying: 
```
TODO: Replace this manual artifact creation with the `log_model` method after `log_model` is released from beta.
```